xchange-bitmax/src/main/java/org/knowm/xchange/bitmax/dto/BitmaxResponse.java

```diff
@@ -0,0 +1,69 @@
+package org.knowm.xchange.bitmax.dto;
+
+import com.fasterxml.jackson.annotation.JsonIgnore;
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class BitmaxResponse<V> {
+
+  @JsonIgnore private final String ac;
+
+  @JsonIgnore private final String accountId;
+
+  @JsonIgnore private final String message;
```

Comment:
for what are those `@JsonIgnore` needed here?

Reply:
I remember, probably this has to do with the response that i was getting. I think that no all of them appear in every request.